src/indexApi.ts

```diff
@@ -1,18 +1,125 @@
 // BASE SETUP
 // =============================================================================
 
+import { asObject, asOptional, asString } from 'cleaners'
 import cluster from 'cluster'
+import { formatISO } from 'date-fns'
 import { forkChildren, rebuildCouch } from 'edge-server-tools'
 
 import { config } from './utils/config'
-import { couchSchema } from './utils/couchSchema'
+import { couchSchema, minAmtDbPrefix } from './utils/couchSchema'
+import { ErrorResponse, makeErrorResponse } from './utils/errorResponse'
+import { mockPlugins } from './utils/mockPlugins'
 
 const express = require('express')
 const http = require('http')
+const nano = require('nano')
+const promisify = require('promisify-node')
+
+const asFulfilledPromise = asObject({
```

Comment:
Rename this. When naming constant, don't use generic terms. Try to be as specific and descriptive as possible.